core_impl/src/test/java/io/opencensus/stats/ViewManagerImplTest.java

```diff
@@ -375,13 +375,13 @@ public void testViewDataWithMultipleTagKeys() {
 
   @Test
   public void testMultipleViewSameMeasure() {
-    View view1 =
+    final View view1 =
```

Comment:
There was a second warning in the file where it wasn't easy to shorten the distance, because the test defined 4 constants at the top of the method (The maximum distance between declaration and use is 3).  I fixed both with `final` for consistency.